charts/kyverno/templates/hooks/pre-delete-configmap.yaml

```diff
@@ -36,7 +36,6 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: {{ template "kyverno.fullname" . }}:remove-configmap
-  namespace: {{ template "kyverno.namespace" . }}
```

Comment:
>namespace is not a valid parameter for a RoleBinding/roleRef field

Can you elaborate why it's not valid?

Reply:
> Can you elaborate why it's not valid?

this is not a valid parameter: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#roleref-v1-rbac-authorization-k8s-io

and If you use a tool to validate manifest in CI (kubeconform for example), it will fail.

Reply:
Ah I see, this is for `roleRef`, not a `Role`